apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -240,6 +239,20 @@ const FapProposalsAndAssignmentsTable = ({
     );
   };
 
+  const handleAssMembersToFapProposals = async (
```

Comment:
The name of this function doesn't sound right 😅

Reply:
Yeah, should've thought more carefully about that one.